content/dstu2/authorization-specification.md

```diff
@@ -0,0 +1,1029 @@
+    specific resource.
+*   Internal errors in the EHR system.
+
+In the majority of cases, the EHR system will have the most information
+as to the cause of the failure, and will be in the best position to 
+offer assistance to the user and/or individuals whom support a given 
+the SMART application.  As such, it is __RECOMMENDED__ that authorization 
+servers and resource servers utilize the "error_uri" parameter as 
+detailed in the following specifications:
+
+*   [OAuth 2.0 - section 4.1.2.1 Error Response](http://tools.ietf.org/html/rfc6749#section-4.1.2.1)
+*   [OAuth 2.0 - section 5.2 - Error Response](http://tools.ietf.org/html/rfc6749#section-5.2)
+*   [Bearer Token Usage - section 3 The WWW-Authenticate Response Header Field](http://tools.ietf.org/html/rfc6750#section-3)
+
+SMART applications that receive such information in error responses 
+__SHOULD__ present such links to end users.  
+
+It is __RECOMMENDED__ that such links provide actionable instructions 
+for either the end user or for individuals whom are supporting the SMART 
+application.  It is __RECOMMENDED__ for SMART applications to provide a 
+mechanism to retry a specific action after an error to assist users in 
+recovering from transient error conditions.
+
+### Security Considerations ###
+*   It is __RECOMMENDED__ that EHRs issue access tokens for short durations 
+    (e.g. ten minutes or less) to minimize the window in which a compromised 
+	token could be excercised.  Otherwise, an EHR __SHOULD__ provide 
+	mechanisms accessible to administrators and end users to revoke tokens 
+	easily in the event of a security incident.
+
+*   Authorization servers __MUST NOT__ utilize the value of the launch code 
+    as a mechanism for passing authenticated state.  Doing so opens the 
+    possibility to a session injection attack, and could open the possibility 
+    of a session fixation attack.
+
+*   SMART applications __MUST__ assure that sensitive information 
+    (authentication secrets, authorization codes, tokens, PHI) are 
+    transmitted ONLY to authenticated servers, over TLS-secured channels.
+	
+*   Resource servers __MUST__ be TLS-secured.
+	
+*   All use of TLS __MUST__ incorporate best current practices for transport 
+    layer security described in [RFC 7525][].  EHRs and SMART applications 
+    __SHOULD__ continue to incorporate updates to current practices and 
+    __SHOULD__ 	document their policies for staying current.
+
+<a id="A"></a>
+	
+## Appendix ##
+
+This section provides Augmented Backus-Naur Form (ABNF) syntax descriptions 
+for the elements defined in this specification using the notation of 
+[RFC 5234][], and additionally utilizes other syntax definitions defined in 
+[Appendix A of the OAuth 2.0 Framework](http://tools.ietf.org/html/rfc6749#appendix-A).
+
+<a id="A.1"></a>
+
+### A.1. Resource Scope Syntax ###
+
+<pre class="terminal">
+scope-name            = resource-context "/" resource-type "." modification-rights
```

Comment:
The = signs in these 5 sections currently align to 3 different columns, could you move A1 left by 2 spaces and A3-5 right by 1 space? :)